pkg/volume/glusterfs/glusterfs.go

```diff
@@ -479,22 +510,128 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 	return nil
 }
 
+func (r *glusterfsVolumeProvisioner) ConstructBitmap() error {
```

Comment:
Bitmap should not be part of the name, since it bleeds out implementation detais of the allocator. need a better name.